.github/ISSUE_TEMPLATE/1-Security-issue.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: "\U0001F46E‍♂️ Security issue"
+name: "\U+1F512 Security issue"
```

Comment:
(This is a change to the Unicode emoji we are using to represent security issues from "Man Police Officer" emoji to "Locked" Emoji which is the more common emoji used for content concerning security)